katdal/chunkstore_s3.py

```diff
@@ -267,14 +277,18 @@ class S3ChunkStore(ChunkStore):
     public_read : bool
         If set to true, new buckets will be created with a policy that allows
         everyone (including unauthenticated users) to read the data.
+    expiry_days : int
+        If set to a value greater than 0 will set a future expiry time in days
+        for any new buckets created.
```

Comment:
I prefer to use `None` for never. Zero sounds like "expire immediately", and since it's not C so we don't have to make some numbers have special meanings.

Reply:
The reason is to make it clear that 0 does nothing, as opposed to explaining that you can't actually use 0 in expiry policies, but no problem to change.